setup.py

```diff
@@ -87,7 +87,7 @@
 ]
 
 tutorials_require = [
-    'scikit-learn>=0.24,<2',
+    'scikit-learn>=0.24,<1.2',
```

Comment:
any reason we changed this?

Reply:
Yes, the tutorials are not working since the demo datasets used are available until 1.2
Since this was for tutorials only I think it was okay to just cap it right away.